hooks/workfiles_management.py

```diff
@@ -31,6 +31,7 @@ class WorkfilesManagement(sgtk.get_hook_baseclass()):
     WORKFILE_ENTITY = "CustomEntity45"
     # The custom entity has the following custom fields:
     # sg_version (number): version number of the scene
+    # sg_name_field (text): user specified name at save time
```

Comment:
what is the difference between this and `CustomEntity45.code` - i am not sure i follow! May be worth explaining.

Reply:
I'll add a comment at the bottom explaining the difference!